esp32/modules/services.py

```diff
@@ -1,59 +1,251 @@
-import uos
-
 # File: services.py
+# Version: 2
+# API version: 1
 # Description: Background services for SHA2017 badge
 # License: MIT
 # Authors: Renze Nicolai <renze@rnplus.nl>
 #          Thomas Roos   <?>
 
-def setup():
+import uos, ujson, easywifi, easyrtc, time, appglue, deepsleep, ugfx, badge, machine
+
+services = [] #List containing all the service objects
+loopCallbacks = {} #Dict containing: {<FUNCTION>:<Wifi required on next run>}
+drawCallbacks = [] #List containing draw functions
+
+def setup(pmCb=False, drawCb=False):
     global services
+    global loopCallbacks
+    global drawCallbacks
+    
+    if pmCb:
+        print("[SERVICES] Power management callback registered")
+        global pmCallback
+        pmCallback = pmCb
```

Comment:
Actually why is this global needed anyway? it doesn't seem to be set to anything else than `power_management` ever - perhaps just call `power_management` instead of `pmCallback`?